examples/stable_diffusion_v2/controlnet.md

```diff
@@ -112,6 +112,70 @@ Here are samples generated by a bird image with DeeplabV3Plus segmentation edge
   <em> Generated Images with ControlNet on DeeplabV3Plus segmentation edge maps </em>
 </p>
 
+## Fine-tuneing using ControlNet
+
+Try to customize your own controlnet? Downloading the SD base weight, parparing your dataset and start training!
```

Comment:
_parparing_, _filefoder_, _orginal_, etc. - please do spell check. I suggest you use tools / plugins for your IDE to automate this work.

Reply:
Received with thanks.